### DISCUSSION TITLE
How do I use my Unraid server as a host with Docker ?

### DISCUSSION THREAD

**Discussion by Logidroid** [ptrumpis/snap-camera-server#168] [Q&A] [answered]
I would like to host docker on my Unraid server but I don't know how to do it :(

![firefox_L9LBZXfvnB](https://github.com/user-attachments/assets/8912f7c9-2918-47c4-9280-6edd2cf6d7cd)

**Comment by ptrumpis** [comment, marked as answer]
I'm not familiar with Unraid, but this might help:
[Make an Unraid package to run server on unraid for local network #74](https://github.com/ptrumpis/snap-camera-server/discussions/74)
